sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- Renamed `projectName` and `deploymentName` tags reported on `QuestionAnsweringClient` activities to `az.cognitivelanguage.project.name` and `az.cognitivelanguage.deployment.name` following OpenTelemetry attribute naming conventions.
```

Comment:
same tags could also apply to `Azure.AI.Language.Conversations`, so it's nice to keep them in `cognitivelanguage` namespace